packages/modules/desktop-viewer-react/README.md

```diff
@@ -47,6 +47,16 @@ export const MyViewerComponent = () => {
 
 ### Props
 
+#### Required
+
+- `enablePerformanceMonitors` - Enable reporting of data from timed events in the iTwin Viewer in order to aid in future performance optimizations. These are the metrics that will be collected and logged to the browser's performance timeline as well as to Azure Application Insights:
+  - Duration of startup to the initialization of iTwin.js services
+  - Duration of startup to the establishment of a connection to the iModel
+  - Duration of startup to the creation of a view state for the iModel
+  - Duration of startup until the last tile is loaded and rendered for the iModel view
```

Comment:
As I understand it, there are an "infinite" number of tiles, eg changing the zoom and LoD will just spawn more tiles to be generated/fetched.  Using the term "last tile" here can be misleading imo.

Reply:
this mark is added in a one time callback when the view opens though, so it technically is the last tile loaded initially (i.e. it will not mark again when resizing or changing the view)

Reply:
maybe add a comment here to say "for the initial iModel view"?

Reply:
updated